server.tsx

```diff
@@ -3,7 +3,7 @@ import { React, ReactDomServer } from './deps.ts';
 import App from './client/app.tsx';
 import { staticFileMiddleware } from './staticFileMiddleware.ts';
 
-const PORT = 3000;
+const PORT = 3333;
```

Comment:
Hey ya'll, is this correct?  The dockerfile exposes port 3000, I think this is causing a 502 error

Reply:
good catch! we are fixing that and a few other things and will be making another pull request shortly.

Reply:
Awesome!  When you make the new PR, request me for review and I'll take a sec to look it over and approve. :)

Reply:
copy! will do